tests/compile.sh

```diff
@@ -62,6 +62,7 @@ else
     source /lustre/f2/dev/role.epic/contrib/Lmod_init.sh
   fi
   # Load fv3 module
+  module purge
```

Comment:
With Gaea, a module purge is very risky and commonly breaks the system. I haven't tested the new C5 nodes, though. I imagine without a specific section for C5, that a purge would still break things at this point and should be avoided.